src/ratelimit.ts

```diff
@@ -257,4 +258,22 @@ export abstract class Ratelimit<TContext extends Context> {
     }
     return res!;
   };
+
+  public resetUsedTokens = async (identifier: string) => {
+    const pattern = [this.prefix, identifier, "*"].join(":");
```

Comment:
I noticed that the reset token tests was not working when I tried the tokenBucket algorithm. It is not able to reset the bucket.

This is because in the tokenBucket algorithm, we use a different key format:

![image](https://github.com/upstash/ratelimit/assets/57228345/0243aba9-4a80-47a4-8e51-163437344a19)

the pattern here doesn't match it.

Reply:
What I mean is, pattern we pass to the scan won't be able to match the keys of the tokenBucket algorithm, since they are not in `prefix:identifier:*` format. They are in `prefix:identifier` format as I show in my ss above.

The pattern doesn't match the entries of the tokenBucket algorithm

Reply:
Okay I found the issue thanks